lib/utils/segment.js

```diff
@@ -37,7 +25,7 @@ function track(payload) {
   if (TRACKING_IS_DISABLED) {
     return BbPromise.resolve();
   }
-  return request('https://api.segment.io/v1/track', payload);
+  return request('https://tracking.serverlessteam.com/v1/track', payload);
```

Comment:
@pmuens This particular path (`/v1/track1) is Segment related. It's the only Lambda proxy set up right now. I could set up a Sentry one as well if needed.

Will ping you about the debug setup.